src/ListView/CreatePage/CreatePageForm.jsx

```diff
@@ -0,0 +1,131 @@
+import React, { useEffect, useState } from "react";
+import { colorTextErrorOnLight } from "@sailthru/design-tokens";
+import {
+  FormGroup,
+  Radio,
+  RadioGroup,
+  Subtext,
+} from "@sailthru/stui-components";
+import { Fieldset, Label, Input } from "@sailthru/stui-elements";
+
+function CreatePageForm({
+  name = "",
+  onChange,
+  isDuplicateName,
+  isNameChanged,
+  setIsValidName,
+}) {
+  const [namingError, setNamingError] = useState("");
+  const [optoutNamingError, setOptoutNamingError] = useState("");
+
+  const checkForNamingError = function () {
+    setNamingError("");
```

Comment:
question: Does `setIsValidName` need to be reset here too?
```suggestion
    setNamingError("");
    setIsValidName(true);
```

Reply:
yes, it should!